sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -549,15 +543,12 @@ def test_user_agent_append(self, resource_group, location, storage_account, stor
 
         def callback(response):
             self.assertTrue('User-Agent' in response.http_request.headers)
-            self.assertEqual(
-                response.http_request.headers['User-Agent'],
-                "azsdk-python-storage-blob/{} Python/{} ({}) customer_user_agent".format(
+            assert ("customer_user_agent azsdk-python-storage-blob/{} Python/{} ({})".format(
```

Comment:
Wouldn't this be considered a breaking change for customers setting a custom UA?

Reply:
Spoke to @lmazuel about this. By design, custom UA are supposed to be appended forward and not at the end. So this is sort of "fixing" storage.